Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
```

Comment:
Really great idea to expand the function of the calculator by adding %.

Reply:
Great formatting for the print statement!

Reply:
Good choice using the f format!

Reply:
I agree, extending the functionality will give a user more options to choose from.